e2e/tests/k8s_api.py

```diff
@@ -496,6 +503,13 @@ def exec_with_kubectl(self, pod, cmd):
                               stdout=subprocess.PIPE,
                               stderr=subprocess.PIPE)
 
+    def patroni_rest(self, pod, path):
```

Comment:
duplicate of [246-252](https://github.com/zalando/postgres-operator/pull/1535/files#diff-2fb572115844fab37e322804b14ff5809f4e3e0f2c12f7add8e5509532403a31R246-R252) ?

Reply:
yes, there's a lot of duplicate code. Don't understand why. I thought to better do it, too :smile: 
Someone should tidy this part up.